CS3230Project/ViewModel/Diagnosis/DiagnosisManagerViewModel.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using CS3230Project.Model.Diagnosis;
+
+namespace CS3230Project.ViewModel.Diagnosis
+{
+    /// <summary>
+    ///   The view model for the diagnosis
+    /// </summary>
+    public static class DiagnosisManagerViewModel
+    {
+
+        /// <summary>
+        /// Adds the diagnosis.
+        ///
+        /// Precondition: none
+        /// Post-condition: the diagnosis provided is added to the database 
+        /// </summary>
+        /// <param name="diagnosisToAdd">The diagnosis to add.</param>
+        /// <returns>
+        ///   True, if the diagnosis was added to the database
+        ///   False, if the diagnosis was not added to the database
+        /// </returns>
+        public static bool AddDiagnosis(Model.Diagnosis.Diagnosis diagnosisToAdd)
+        {
+            return DiagnosisManager.AddDiagnosis(diagnosisToAdd);
+        }
+
+        /// <summary>
+        /// Modifies the diagnosis.
+        ///
+        /// Precondition: none
+        /// Post-condition: the provided diagnosis is modified with the provided details
+        /// </summary>
+        /// <param name="modifiedDiagnosis">The modified diagnosis.</param>
+        /// <returns>
+        ///   True, if the diagnosis was modified
+        ///   False, if the diagnosis was not modified
+        /// </returns>
+        public static bool ModifyDiagnosis(Model.Diagnosis.Diagnosis modifiedDiagnosis)
```

Comment:
Currently there is no way to modify the diagnosis. The ModifyDiagnosis method is not even used.

Reply:
You are correct. I had the modify methods for my initial setup. After updating my implementation, I wasn't sure if we would have the functionality to modify a diagnosis since they are part of the medical history. I left the methods in case we wanted to be able to edit.

Reply:
Fixed.